modules/nix/default.nix

```diff
@@ -523,12 +523,14 @@ in
             auto-optimise-store = mkOption {
               type = types.bool;
               default = false;
-              example = true;
               description = ''
                 If set to true, Nix automatically detects files in the store that have
                 identical contents, and replaces them with hard links to a single copy.
                 This saves disk space. If set to false (the default), you can still run
-                nix-store --optimise to get rid of duplicate files.
+                {command}`nix-store --optimise` to get rid of duplicate files.
+
+                NOTE: This is known to cause Nix Store corruption and using
+                {option}`{nix.gc.automatic} instead is recommended.
```

Comment:
Should probably be a `::: {.warning}`, but also if this is getting fixed then we might not need to warn explicitly against it in the `description` – the assertion should do a good enough job by itself.

Reply:
Removed, also `::: {.warning}` didn't do anything when I tested it (neither in the manpages or the HTML manual)